src/plugins/intel_gpu/src/kernel_selector/kernels/rms/rms_kernel_bfyx_opt.cpp

```diff
@@ -120,7 +122,7 @@ RMSKernelBase::DispatchData RMSKernelBfyxOpt::SetDefault(const rms_params& param
 
         dispatchData.itemsNum = dispatchData.dataSize;
```

Comment:
Hint: maybe check if dispatchData.dataSize is actually constant and won't change when executing LLM, therefore we know beforehand what will always dispatchData.itemsNum be in each execution

Reply:
In Qwen, dataSize is always equal to 896.

Reply:
I see, so that looks to me like a possible optimization there

Reply:
And also you would know LWS and therefore you could use reqd_work_group_size

Reply:
Also 896 seems relatively not a not. It makes sense why reducing stack size would improve performance, since you are basically freeing up resources that are not used.